monasca/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for Monasca running in Kubernetes
 name: monasca
-version: 0.3.4
```

Comment:
you might have to do this in stages as i dont think the ci job would be happy

Reply:
I think it should work, yeah? We can try and see :smiley: